src/blocked.c

```diff
@@ -110,7 +110,7 @@ void updateStatsOnUnblock(client *c, long blocked_us, long reply_us){
     if (!(c->lastcmd->flags & CMD_SKIP_SLOWLOG)) {
         slowlogPushEntryIfNeeded(c,c->argv,c->argc,total_cmd_duration);
         /* Log the reply duration event. */
-        latencyAddSampleIfNeeded("command-unblocking",reply_us/1000);
+        latencyAddSampleIfNeeded("command-unblocking",total_cmd_duration/1000);
```

Comment:
as discussed before, the latency monitor is about inducing latency for other tasks in redis, each time we block the main thread for long, we need to report it in the latency monitor.
so in this case we block it only for the ducation of the reply, i think you should revert the last change and use `reply_us`, not `total_cmd_duration`

Reply:
Agreed @oranagra . Completely missed it. Fixed in the new commit